clustering/weld/core/src/main/java/org/wildfly/clustering/weld/bean/proxy/util/UtilProxyBeanMarshallerProvider.java

```diff
@@ -7,25 +7,17 @@
 
 import org.jboss.weld.bean.proxy.util.SerializableClientProxy;
 import org.jboss.weld.serialization.spi.BeanIdentifier;
-import org.wildfly.clustering.marshalling.protostream.ProtoStreamMarshaller;
-import org.wildfly.clustering.marshalling.protostream.ProtoStreamMarshallerProvider;
+import org.wildfly.clustering.marshalling.protostream.AbstractSerializationContextInitializer;
+import org.wildfly.clustering.marshalling.protostream.SerializationContext;
 import org.wildfly.clustering.marshalling.protostream.reflect.BinaryFieldMarshaller;
 
 /**
  * @author Paul Ferraro
  */
-public enum UtilProxyBeanMarshallerProvider implements ProtoStreamMarshallerProvider {
-
-    SERIALIZABLE_CLIENT_PROXY(new BinaryFieldMarshaller<>(SerializableClientProxy.class, BeanIdentifier.class, String.class, SerializableClientProxy::new)),
-    ;
-    private final ProtoStreamMarshaller<?> marshaller;
-
-    UtilProxyBeanMarshallerProvider(ProtoStreamMarshaller<?> marshaller) {
-        this.marshaller = marshaller;
-    }
+public class UtilProxyBeanMarshallerProvider extends AbstractSerializationContextInitializer {
```

Comment:
The `UtilProxyBeanMarshallerProvider` is missing `@MetaInfServices(SerializationContextInitializer.class)` and is thus is never registered nor used.

Reply:
Good catch.  This is a rebase mishap of https://github.com/wildfly/wildfly/pull/17896